layouts/Error.tsx

```diff
@@ -0,0 +1,79 @@
+// CSS
+import styles from '../styles/layouts/Error.module.scss';
+// Components
+import NavBar from '../components/NavBar';
+import Button, { ButtonType } from '../components/Button';
+import React from 'react';
+import { useEffect, useState } from 'react';
+import Footer from '@/components/Footer';
+
+const departure_url = 'https://tcsca';
+
+export default function ErrorLayout({
+  children,
+}:{
+  children: React.ReactNode
+}) {
+  const [allowReport, setAllowReport] = useState(false);
+  useEffect(() => {
+    // If the user is coming from a broken link on our site, they can report an issue
+    if (document.referrer.includes('localhost:3000' || document.referrer.startsWith(departure_url))){
```

Comment:
```suggestion
    if (document.referrer.includes('localhost:3000' || document.referrer.startsWith(departure_url))) {
```

I'm surprised the formatter did not fix that. Just a style nit though.